docs/content/latest/admin/yb-ts-cli.md

```diff
@@ -0,0 +1,229 @@
+## Command line help
+
+To display the available online help, run `yb-ts-cli` without any commands or options (flags) at the YugabyteDB home directory.
+
+```sh
+$ ./bin/yb-ts-cli
+```
+
+## Commands
+
+##### are_tablets_running
+
+If all tablets are running, returns "All tablets are running".
+
+**Syntax**
+
+```sh
+yb-ts-cli [ --server_address=<host>:<port> ] are_tablets_running
+```
+
+- *host*:*port*: The *host* and *port* of the tablet server. Default is `localhost:9100`.
+
+##### count_intents
+
+Print the count of uncommitted intents (or [provisional records](../../../architecture/transactions/ditributed-txns/#provisional-records)). Useful for debugging transactional workloads.
+
+**Syntax**
+
+```sh
+$ ./bin/yb-ts-cli  [ --server_address=<host>:<port> ] count_intents
+```
+
+- *host*:*port*: The *host* and *port* of the tablet server. Default is `localhost:9100`.
+
+##### current_hybrid_time
+
+Prints the value of the current [hybrid time](../../../architecture/transactions/single-row-transactions/#hybrid-time-as-an-mvcc-timestamp).
+
+**Syntax**
+
+```sh
+$ ./bin/yb-ts-cli  [ --server_address=<host>:<port> ] current_hybrid_time
+```
+
+- *host*:*port*: The *host* and *port* of the tablet server. Default is `localhost:9100`.
+
+##### delete_tablet
+
+Deletes the tablet with the specified tablet ID (`tablet_id`) and reason.
+
+**Syntax**
+
+```sh
+$ ./bin/yb-ts-cli  [ --server_address=<host>:<port> ] delete_tablet <tablet_id> "<reason-string>"
+```
+
+- *host*:*port*: The *host* and *port* of the tablet server. Default is `localhost:9100`.
+- *tablet_id*: The identifier (ID) for the tablet.
+- *reason-string*: Text string providing useful information on why the tablet was deleted.
+
+##### dump_tablet
+
+Dump, or export, the specified tablet ID (`tablet_id`).
+
+**Syntax**
+
+```sh
+yb-ts-cli [ --server_address=<host>:<port> ] dump_tablet <tablet_id>
+```
+
+- *host*:*port*: The *host* and *port* of the tablet server. Default is `localhost:9100`.
+- *tablet_id*: The identifier (ID) for the tablet.
+
+##### list_tablets
+
+Lists the tablets on the specified tablet server, displaying the following information:
+
+- Tablet id
+- State
+- Table name
+- Partition
+- Schema
+- properties
+  - lumn_name
+  - Tablet id
+  - State
+  - Table name
+  - Partition
+  - Schema
+
+**Syntax**
+
+```sh
+yb-ts-cli [ --server_address=<host>:<port> ] list_tablets
+```
+
+- *host*:*port*: The *host* and *port* of the tablet server. Default is `localhost:9100`.
+
+##### set_flag
+
+Sets the specified configuration option (flag) for the tablet server.
+
+**Syntax**
+
+```sh
+$ ./bin/yb-ts-cli [ --server_address=<host>:<port> ] set_flag [ --force ] <flag> <value>
+```
+
+- *host*:*port*: The *host* and *port* of the tablet server. Default is `localhost:9100`.
+- --force: Flag to to allow a change to a flag (option) that is not explicitly marked as runtime-settable. The change may be ignored on the server or may cause the server to crash. See [--force](#force).
```

Comment:
Where did we get this part from `may cause the server to crash`? I don't think that should be, unless we have some bad code that behaves badly on certain args, but also does not validate them...

Reply:
Perhaps we should at least put some note here, so it doesn't sound scary. Something like, `may cause the server to crash, if unsafe values are provided` ? That sounds a bit more defensive, even if we still do not define what is safe and what is not.

In practice, our code should already be defensive or have flag validation.

Reply:
Changed to say "Note that the change may be ignored on the server or may cause the server to crash, if unsafe values are provided."